NameAndSignatureCapture/force-app/main/default/lwc/signatureDemo/signatureDemo.css

```diff
@@ -0,0 +1,16 @@
+/*
+ * For demo purposes, here we are defining a font as base64 string to be used by signaturePad component.
+ * When using signaturePad component in your projects, you can provide your own font however you like.
```

Comment:
You may (somewhere) want to reference that changes to this font would require changes to the font sniffing logic in `signatureDemo.js` too.